docs/pim/catalog_api.md

```diff
@@ -0,0 +1,43 @@
+---
+description: Use PHP API to get information about and manage product catalogs.
+---
+
+# Catalog API
+
+To get information about product catalogs and manage them, use `CatalogServiceInterface`.
+
+## Get catalog
+
+To get a single catalog, you can use `CatalogServiceInterface::getCatalog()` and provide it with catalog ID, or `CatalogServiceInterface::getCatalogByIdentifier()` and pass the identifier:
+
+``` php
+[[= include_file('code_samples/api/product_catalog/src/Command/CatalogCommand.php', 70, 72) =]]
+```
+
+## Get products in catalog
+
+To get products from a catalog, request the product query from the catalog object  with `catalog::getQuery()`.
```

Comment:
Did you mean `\Ibexa\Contracts\ProductCatalog\Values\CatalogInterface::getQuery` ?